### DISCUSSION TITLE
CMS Kit testing and usage

### DISCUSSION THREAD

**Discussion by pvaz** [abpframework/abp#7075] [General]
Hi,

I cannot find a tutorial or explanation on how I can add the cms-kit module and use it in Abp Commercial or even on the Abp Framework.

I know it's under development for next major release but even I cannot test it on the preview version 4.2, any tutorial about this also ?

In my first instance I need a way to be able to have a startup page that is a web page with a login link and after login successfuly, to show the backoffice.

Normally the first page already contains the menus and I want to be a page defined by me and only after login to show the normal view of the system and functionalities.

Any help was great.

Paulo

**Comment by cotur** [comment]
Hi @pvaz,

Cms-Kit is published with some features but not documented yet. We are working to improve it and adding new features a lot on following development cycles.

As the write in the [blog](https://blog.abp.io/abp/ABP.IO-Platform-v4.1-RC-Has-Been-Released) post, you will get a sample public application in startup template (commercial), there will be a lot of usages of Cms-Kit components.

We are highly focused on development for now. Whenever the Cms-Kit base components become stable, we will start to documenting it.

**Comment by pvaz** [comment]
Hi @cotur ,

Thank you for your quick reply.

As a recent user of abp commercial, i'm still looking around how to move and make things.
Even there is lot of documentation, I still have problems when I try to move forward.

I know that under nightly build it's possible to access the preview of cms-kit, but when trying to switch to nightly-build, it gives error.

It was indeed perfect to have a demo version, even simples of the project that as the cms-kit working as it is. This way I could see how's it intereact and how he can be used.

Another issue and question is how can it be possible for me to add the current preview module of cmskit to a 4.1 project for instance ?

Because when I try to add the module using abp add-module, it says it does not exist even it exists on the modules section of version 4.1.

Thanks,
Paulo
